Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "totp"
-version = "0.3.0"
+version = "0.4.1"
```

Comment:
nitpick: how come it's not just 0.4.0?

I know this was changed in each commit, but I doubt we want to publish any of those separately, just the newest state once merged